cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/Partitioning.java

```diff
@@ -88,6 +88,11 @@ public FieldType getFieldType(String fieldName) {
     return fields;
   }
 
+  @Override
+  public String toString() {
+    return fields.toString();
```

Comment:
Shall we use the standard toString implementation that has the class name in it?